packages/sql-mapper/test/telemetry.test.js

```diff
@@ -4,14 +4,24 @@ const { test } = require('node:test')
 const { deepEqual, equal, match, ok } = require('node:assert')
```

Comment:
Are there any actual change to this file minus formatting? Why did you prettier almost everywhere?